charts/argo-templates/templates/ingest-file.yaml

```diff
@@ -17,6 +17,8 @@ spec:
           - name: timeout
           - name: sa-secret
           - name: sa-secret-key
+          - name: sa-secret-volume
+          - name: volume-mount-path
```

Comment:
I would rather not have extra arguments for this. The choice of value is fully-contained within each task step (k8s generates a volume from the secret, and volume-mount-path is always == sa-secret-key). Having the Helm variables was just for convenience.

Reply:
Okay, sounds good 👍That simplifies things then